src/components/Labelled/Labelled.tsx

```diff
@@ -48,6 +57,17 @@ export default function Labelled({
     </div>
   ) : null;
 
+  const characterCountMarkup = showCharacterCount &&
+    maxLength && (
+      <span
+        id={`${id}-labeled-character-counter`}
+        className={styles.CharacterCount}
+        aria-live="polite"
```

Comment:
This looks great, @jfangrad!

I tested with a few common screen reader/browser combos and the character count help text is read when the field gets focus, and the updates get read. One small recommendation I'd make is to add `aria-atomic="true"` to the `aria-live` container here. Right now, NVDA with Firefox is only reading the updated character count, not the whole string, so the context is not quite as clear. Adding that attribute will cause the whole `<span>` to be read when it changes.

Reply:
Bumping @dpersing's suggestion to add `aria-atomic="true"` to the `aria-live` container.